src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -60,6 +60,19 @@
     public static final EditCommand.EditPersonDescriptor DESC_AMY;
     public static final EditCommand.EditPersonDescriptor DESC_BOB;
 
```

Comment:
I believe these should be placed under testutil instead (not logic/commands/CommandTestUtil.

Also, it will be great if we tested invalid dates and cheese types as well.

Reply:
2 Things
1. This is to test the `EditPersonDescriptor` (which is not a part of the model, but only used inside `Commands`, hence this should be in the right place
1. Regarding invalid dates, will add a few in the Dates class
1. Regarding invalid cheese types, currently there is no way for a cheese to be of an invalid type (because it just needs a name), unless we check for empty String

Reply:
Will be adding test cases for empty String